theano/gpuarray/type.py

```diff
@@ -470,7 +471,17 @@ def c_headers(self):
                 '<gpuarray_api.h>']
 
     def c_header_dirs(self):
-        return [pygpu.get_include(), numpy.get_include()]
+        other_dirs = []
+        alt_inc_dir = os.path.abspath(os.path.normpath(sys.exec_prefix + '/Library/include'))
```

Comment:
On Linux, headers seem to be in `sys.exec_prefix + '/include'` instead.
Same for `lib`, by the way. The whole `/Library` in the path does not seem to be there.